scripts/compare_disconnect.py

```diff
@@ -1,47 +1,77 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 
-import urllib.request, json
+import requests
+import json
 import argparse
 
-parser = argparse.ArgumentParser(description='Compare Shavar and Disconnect.me blacklist.')
-parser.add_argument("-f", "--file", help="blacklist to verify")
+parser = argparse.ArgumentParser(description='Compare Shavar and Disconnect.me blocklist.')
+# parser.add_argument("file", type=str, help="blacklist to verify")
```

Comment:
Yes, because instead of specifying the file in the command I just call it in the code https://github.com/mozilla-services/shavar-prod-lists/pull/369/files#diff-6f394a1596fd459ef16c67d6ca6cb0b40f1fbb27cfc831292c923b340aa2d9c1R62